docs/topics/forms/media.txt

```diff
@@ -2,9 +2,9 @@
 Form Assets (the ``Media`` class)
 =================================
 
-Rendering an attractive and easy-to-use Web form requires more than just
+Rendering an attractive and easy-to-use web form requires more than just
 HTML - it also requires CSS stylesheets, and if you want to use fancy
-"Web2.0" widgets, you may also need to include some JavaScript on each
+"web2.0" widgets, you may also need to include some JavaScript on each
```

Comment:
This should remain capitalized:
```suggestion
"Web 2.0" widgets, you may also need to include some JavaScript on each
```

Reply:
Does anyone still use the phrase web 2.0? Could we chop it?

Reply:
Agreed, we can chop it.